src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  moviesList: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ moviesList }) => (
+  <>
+    <div className="movies">
+      {moviesList.map(movie => (
+        <div key={movie.imdbId} className="card" data-cy="Movie">
+          <MovieCard
+            title={movie.title}
+            description={movie.description}
+            imageSource={movie.imgUrl}
+            imageBSource={movie.imdbUrl}
```

Comment:
pass the whole movie object and use destructure it in `MovieCard` component 
```suggestion
          movie={movie}
```